site/faq.md

```diff
@@ -64,6 +64,13 @@ This
 [flux (daemon) operator](https://github.com/justinbarrick/flux-operator)
 project may be of use for managing multiple daemons.
 
+### Do I have to have my application code and config in the same git repo?
```

Comment:
I feel that the preceding question should perhaps reference this one and/or re-iterate that when we are talking about "one git repo" we mean "one git repo for all your config", not "one git repo for all your applications".

Reply:
Good thought :+1: